docs/en/users/06_Fever_API.md

```diff
@@ -26,10 +26,10 @@ Tested with:
 - iOS
   - [Fiery Feeds](https://itunes.apple.com/app/fiery-feeds-rss-reader/id1158763303)
   - [Unread](https://itunes.apple.com/app/unread-rss-reader/id1252376153)
-  - [Reeder-4](https://itunes.apple.com/app/reeder-4/id1449412357)
+  - [Reeder 4](https://itunes.apple.com/app/reeder-4/id1449412357) when using self-hosted Fever option
```

Comment:
I have not tried myself, but assuming it is possible, we should have a note telling to favour the Google  Reader API option

Reply:
I've been using it for years and it works great. There is one bug in the current update - the native FreshRSS implementation, if I add just the host, it's converting to full URL with /r/api/greader.php which does not work. I already wrote an email to fix this.

So Fever still works, it's just that the native uses greader.

Reply:
Reeder's developer has promised a fix for the URL autocompletion.

Reply:
I'm trying "FreshRSS implementation" in Reeder4 since yesterday, and I have no major problems to report!

I confirm tha Reeder convert itself URL!
But on my version, it convert "domaine.com" in `http://domain.com/p/api/greader.php`
Note that for me is /**p**/api//greader.php and not  /**r**/api/greader.php

A problem comes from the fact that it adds "http" and not "https", which can lead to a refusal (depending on the server configuration).
Mine refused while it is set to redirect by itself. 

But for the rest, I have no problems to report. 
Titles, texts, images and links can be loaded without any problems.
The categories are respected and I can edit my feeds.